lib/ui/nav_pages/home_page/widgets/home_list_items.dart

```diff
@@ -15,17 +15,37 @@ import '../home_page_viewmodel.dart';
 
 final navigationService = locator<NavigationService>();
 
-class ThreadTextAndIcon extends StatelessWidget {
+class ThreadTextAndIcon extends ViewModelWidget<HomePageViewModel> {
   const ThreadTextAndIcon({Key? key}) : super(key: key);
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, viewModel) {
     return _TextAndIcon(
```

Comment:
error • The named parameter 'actionsAlignment' isn't defined • lib/ui/view/draft/app/show_alert_dialog_widget.dart:50:5 • undefined_named_parameter

Reply:
I'ts actually defined from its source code when I checked and its throwing no error, I really don't understand what's going on. the class AlertDialog, actually gives a defined parameter actionsAlignment

Reply:
Resolved